### ISSUE TITLE
Output Path Changed on Rename

### ISSUE THREAD

**Issue by AdamCatalyst** [Sappharad/MultiPatch#20]
A minor annoyance. When outputting file, the moment that you set the output file name, the output path changes to the users Documents folder. If the user manually sets the output path first, then sets the output filename, the user selected path is ignored, and the Documents folder is used instead.

**Comment by Sappharad** [comment]
I've followed your instructions and it works fine for me. See video below. Am I doing something wrong?

https://user-images.githubusercontent.com/1350664/231614284-c2999e42-ef6e-40bb-a7bd-42265386277a.mp4

**Comment by AdamCatalyst** [comment]
Yes, the video is consistent with what I see, that's not the problem. Sorry, I was unclear, if not outright misleading. I will provide a video myself to explain better. 

Correction: The issue occurs when a custom path is not specified. 

The moment the file is named, the path switches to Documents, rather than staying on the currently selected path as ne would expect.

https://user-images.githubusercontent.com/2363378/231655298-a13ae28a-b601-4f44-9b0b-0fb05c07b9f7.mp4

**Comment by Sappharad** [comment]
I'm confused by your video. Aren't you doing the exact same thing I was? It looks like as soon as you start typing in the save dialog the folder switches. But when I started typing in the same place in my video it does not. Even if I don't switch folders first, (which is the only difference I can see) I stay in the folder where I started. It does not change folders on me.

I'm wondering if it has anything to do with your desktop being stored in iCloud. I don't customize the behavior of the save dialog at all other than providing the file extension, it's the standard save dialog provided by macOS. If you were to move the other two files somewhere that isn't backed up to iCloud does the same problem happen?
